src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -590,60 +591,93 @@ private void inviteChatMember(XmppChatMember chatRoomMember,
         }
     }
 
-    private BridgeSession inviteParticipant(
-            Participant participant,
-            boolean reInvite,
-            boolean[] startMuted)
+    /**
+     * Selects a {@link BridgeState} to use for a specific {@link Participant}.
```

Comment:
i'm confused by this, selecting a 'bridge state' for a participant seems weird.  isn't it a bridge we're selecting?

Reply:
and what is the relationship between this and the 'doSelect' for bridges above?

Reply:
The naming is not ideal (suggestions for improvement are welcome). BridgeState is the global state of a jitsi-videobridge instance that jicofo knows about (one BridgeState object per jitsi-videobridge instance). So this is what we're selecting. I've improved the documentation in BridgeState.java which explains what it is.
The selectBridge() method in JitsiMeetConferenceImpl contains some conference-specific logic for the selection (the "enforced" part), and failure handling.

Reply:
could it just be named `Bridge`?